python-backend/app/api/applications/resources/application.py

```diff
@@ -22,24 +22,35 @@
 class ApplicationListResource(Resource, UserMixin):
     parser = reqparse.RequestParser()
     parser.add_argument(
-        'application_no', type=str, required=True, help='Number of the application being added.')
+        'application_no',
+        type=str,
+        required=True,
+        help='Number of the application being added.',
+        location='form')
```

Comment:
What's the story behind these application changes? Seems unrelated to the rest of the PR

Reply:
This was me updating the previous endpoints I had worked on to bring them inline with some new information we discovered.

Reply:
Okay. I'm marking it as resolved. With a mammoth PR like this, it's probably best to split out these changes if they're not 100% related to the primary PR goal